src/Dapr.AspNetCore/Dapr.AspNetCore.csproj

```diff
@@ -11,6 +11,10 @@
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <!-- Adding ref. to System.IO.Pipelines explicitly to work around build issue with .netcore sdk 3.1.102
+    https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1-known-issues.md#net-core-312
+    This should be reverted when the fix will be available in future .net core sdk. -->
+    <PackageReference Include="System.IO.Pipelines" Version="4.7.0" />
```

Comment:
Yeah, this bug is pretty unfortunate. Hopefully it didn't take too long to figure out.